EstateAdministrationUI.IntegrationTests/Common/Setup.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace EstateAdministrationUI.IntegrationTests.Common
+{
+    using System;
+    using Ductus.FluentDocker.Services;
+    using Ductus.FluentDocker.Services.Extensions;
+    using NLog;
+    using Shared.Logger;
+    using Shouldly;
+    using TechTalk.SpecFlow;
+
+    [Binding]
+    public class Setup
+    {
+        protected Setup()
+        {
+
+        }
+
+        protected static IContainerService DatabaseServerContainer;
+
+        public static INetworkService DatabaseServerNetwork;
+
+        public static String SqlServerContainerName = "shareddatabasesqlserver";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'SqlServerContainerName' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=5485852)